Workbooks/WorkbooksMetadata.json

```diff
@@ -544,7 +544,14 @@
     "title": "Security Status",
     "templateRelativePath": "SecurityStatus.json",
     "subtitle": "",
-    "provider": "Microsoft"
+    "provider": "Microsoft",
+    "author": {
```

Comment:
Hey @erant10, according to Workbook metadata format we are not adding author, support and categories in this file, can you please check if it is needed, thanks.

Reply:
updated workbooks metadata to support new properties